.circleci/config.yml

```diff
@@ -151,33 +112,77 @@ jobs:
       - run:
           name: execute sonar-scanner
           command: >
-            sonar-scanner \
-              -Dsonar.login="${SONAR_TOKEN}" \
-              -Dsonar.projectVersion="${CIRCLE_BUILD_NUM}" \
-              -Dsonar.branch="${CIRCLE_BRANCH}"
+            if [ ! -z "$SONAR_TOKEN" ] && \
+              [ ! -z "$CIRCLE_BUILD_NUM" ] && \
+              [ ! -z "$CIRCLE_BRANCH" ]; then
+              sonar-scanner \
+                -Dsonar.login="${SONAR_TOKEN}" \
+                -Dsonar.projectVersion="${CIRCLE_BUILD_NUM}" \
+                -Dsonar.branch="${CIRCLE_BRANCH}"
+            else
+              echo "required env vars not found"
+            fi
       - save_cache:
           key: sonar-{{ epoch }}
           paths:
             - ~/.sonar
 
 
-  # dockerize:
-  #   <<: *defaults
-  #   machine: true
-  #   steps:
-  #     - attach_workspace:
-  #         at: /opt/iroha/build
-  #     - run: TODO
-  #         name: todo
-  #         command: echo "I am happy albatross"
-
+  macos-build:
+    #docker:
+    #  - image: postgres:9.5
+    #  - image: redis:3.2.8
```

Comment:
why commented?

Reply:
Should I remove comments?

Reply:
I mean why those containers isn't launched?

Reply:
MacOS builds for circle ci 2.0 are bugged. When I uncomment it, circle ci thinks that I want to start linux, not mac... Everything fails.